.travis.yml

```diff
@@ -1,4 +1,5 @@
 language: go
 
 go:
-  - tip
+  - 1.9.x
+  - 1.10.x
```

Comment:
You need to put quotes around this, so it doesn't get interpreted as go 1.1.
https://docs.travis-ci.com/user/languages/go/

Alternatively, just remove this line and use 1.9

Reply:
Not true - see the build page for this commit correctly building against 1.9 and 1.10 https://travis-ci.org/google/martian/builds/377094519?utm_source=github_status&utm_medium=notification.

The other projects I contribute similarly use this notation and we've seen no problems with it.